src/components/Question.tsx

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Why was this added? Eslint isn't flagging anything without for me. You also don't enable this rule again anywhere. Could we also try to stick these as close as possible, or even inline with the function they affect to convey intent? (See line 49 for example).

Reply:
Oh, looks like VSCode added it.